src/app.zig

```diff
@@ -1365,6 +1365,23 @@ pub const App = struct {
                             },
                         );
                     }
+                    switch (mouse.button) {
+                        .left => {
+                            if (mouse.type == .press) {
+                                const buffer = self.selectedBuffer() orelse @panic("no buffer");
+                                const command = "/query ";
+                                const content: []const u8 = std.mem.concat(self.alloc, u8, &[_][]const u8{ command, user.nick }) catch |err| {
+                                    log.err("couldn't allocate memory: {}", .{err});
+                                    return;
+                                };
+                                self.handleCommand(null, buffer, content) catch |err| {
+                                    log.err("couldn't handle command: {}", .{err});
+                                };
+                                self.state.mouse.?.type = .release;
```

Comment:
Why are you changing the state of the saved mouse event? This has a codesmell...

Reply:
This smells but it's necessary. Otherwise, when you click on a name, in the new buffer another name could appear in that location and you would then open that buffer. This will continue until no other name is at that location or the name is the same

Reply:
That makes sense, and I've found other places where I'm manually changing the mouse state too. I'm fine keeping it smelly for now, someday (tm) we may have to refactor mouse handling a bit to be cleaner.